src/components/textarea/template.njk

```diff
@@ -1,7 +1,13 @@
 {% from "label/macro.njk" import govukLabel %}
 
-{{- govukLabel(classes, labelText, hintText, errorMessage, id) -}}
+{{- govukLabel({
+  "labelText": params.label.labelText,
+  "hintText": params.label.hintText,
```

Comment:
we might want to use safe here to avoid encoding?
`"hintText": params.label.hintText | safe`
this might be a valid question for most of the text content

Reply:
I think we need to take a look at how we use safe… we're really inconsistent at the minute. I don't see the harm in escaping HTML entities.

Reply:
Yes, we should act consistently on this matter.